### DISCUSSION TITLE
selection of proper batch size

### DISCUSSION THREAD

**Discussion by saeedshahi** [mir-group/nequip#226] [Q&A] [answered]
Hi,
I saw in configs/full.yaml file, you suggested a small batch size between 1-5 in case of energies and forces training and selecting a bigger one in case of just energies training.
Is there any (maybe statistical or ...) reason for that? or it is a hyper parameter and we just should confirm that by testing?

Best,
Saeed

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @saeedshahi ,

Yes, it is basically just a hyperparameter. The logic behind that choice is the number of labels: when you are doing force training, each frame has 3N + 1 labels (3N force components and 1 total energy), whereas when you do just energy training, each frame only contributes a single label.
